src/tests/functional/plugin/shared/src/low_precision_transformations/add_transformation.cpp

```diff
@@ -53,11 +53,20 @@ std::string AddTransformation::getTestCaseName(const testing::TestParamInfo< Add
 }
 
 void AddTransformation::SetUp() {
+    abs_threshold = 1.1;
+    rel_threshold = 3;
     ngraph::element::Type precision;
     ngraph::PartialShape inputShape;
     AddTestValues param;
     std::tie(precision, inputShape, targetDevice, param) = this->GetParam();
 
+    ov::PartialShape inputShape2 = inputShape;
+    if (param.broadcast) {
+        inputShape2[2] = 1;
+        inputShape2[3] = 1;
+    }
+    init_input_shapes({ inputShape, inputShape2 });
+
     function = ngraph::builder::subgraph::AddFunction::getOriginal(
```

Comment:
@itikhono,
There are helper functions in src/tests/ov_helpers/ov_lpt_models. Should it be fixed ngraph -> ov namespace in this PR or it could be done in next separate PR? There are already 200+ files in this PR

Reply:
Talked offline. This will be done in a next separate PR since there are already too many files in this PR

Reply:
created [ticket](https://jira.devtools.intel.com/browse/CVS-129934)